src/Numerics/DeterminantOperators.h

```diff
@@ -318,7 +318,7 @@ inline void mw_InverseUpdateByColumn(const int nw,
                                      const RefVector<OffloadVector<T>>& rvec_list,
                                      const RefVector<OffloadVector<T>>& rvecinv_list,
                                      int colchanged,
-                                     std::vector<T> ratiolist)
+                                     OffloadVector<T> ratiolist)
```

Comment:
It would be better to pass this vector by reference, and it could probably be const (`const OffloadVector<T>& ratiolist`)